### ISSUE TITLE
Error using base64 encoded s3 credentials while performing backup

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#28543]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**

- Create a repo - https://github.com/mrajashree/charts and branch - backup-restore
- Deploy app - backup-restore-operator from this catalog
- Create resourceset using kubectl
- Create a secret using Dashboard
- Create a backup from `resources.cattle.io`.
- Give resourceset name and name of the backup.
- Backup gets deployed successfully.
- Error in `backup-restore-operator`
```
getting creds secret creds from default
INFO[2020/08/25 01:36:09] invoking set s3 service client                s3-accessKey="<>=" s3-bucketName=<>t s3-endpoint=<> s3-endpoint-ca= s3-folder=ecm1 s3-region=us-west-2
ERRO[2020/08/25 01:36:10] error syncing 'default/s3-backup-demo': handler backups: failed to check s3 bucket:<>, err:400 Bad Request, requeuing
INFO[2020/08/25 01:36:10] Processing backup s3-backup-demo
```

**Result:**
Error should not be displayed in the logs. Backup should successfully get created in S3

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): master-head - commit id: `56b819995`
- Installation option (single install/HA): HA

**Comment by mrajashree** [comment]
Fix is available with v0.0.1-rc4
Set `tag: v0.0.1-rc4` in values.yaml while upgrading the chart for backup-restore-operator

**Comment by sowmyav27** [comment]
**Verified on master-head commit id: `9b6984d5f` using `tag: v0.0.1-rc4`**

- Create a repo - https://github.com/mrajashree/charts and branch - backup-restore
- Deploy app - backup-restore-operator from this catalog
- Create resourceset using kubectl
- Create a secret using Dashboard in `cattle-resources-system` ns
- Create a backup from resources.cattle.io.
- Give resourceset name and name of the backup.
- Backup gets deployed successfully.
- Backup gets created successfully in S3.
- User is able to restore from this backup.
